test/runtests.jl

```diff
@@ -1,4 +1,5 @@
 using OpenCV
 using Base.Test
+using Images
```

Comment:
It would be preferable that you first test the new generic Mat constructors with `OpenCV.jl` native methods.  Once completed you can submit a separate PR with support for `Images` and any other relevant packages.